plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/BasePolicyDataProvider.java

```diff
@@ -29,32 +30,37 @@
  * The general logic catch policy value in .ini file as system property next in
  * windows registry under HKEY_CURRENT_USER and HKEY_LOCAL_MACHINE nodes.
  */
-public class BasePolicyDataProvider {
+public class BasePolicyDataProvider implements DBPPolicyDataProvider {
     private static final Log log = Log.getLog(BasePolicyDataProvider.class);
 
     private static final String DBEAVER_REGISTRY_POLICY_NODE = "Software\\DBeaver Corp\\DBeaver\\policy"; //$NON-NLS-1$
-    private static final BasePolicyDataProvider INSTANCE = new BasePolicyDataProvider();
+
+    public static final String POLICY_DATA_EXPORT = "policy.data.export.disabled"; //$NON-NLS-1$
+    public static final String POLICY_DATA_COPY = "policy.data.copy.disabled"; //$NON-NLS-1$
+
+    private static BasePolicyDataProvider INSTANCE = new BasePolicyDataProvider();
```

Comment:
`INSTANCE` -> `instance` cuz not the constant. 
+ Maybe mark `instance` as `volatile` and synchronize' setInstance'?

Reply:
`setInstance` is called only once on a really early app start. I don't see a reason for adding synchronisation.